frontend/tests/controllers-test/challengeListCtrl.test.js

```diff
@@ -0,0 +1,282 @@
+'use strict';
+
+describe('Unit tests for Challenge List Controller', function () {

```

Comment:
`Unit tests for Challenge List Controller` -> `Unit tests for challenge list controller`

Reply:
@RishabhJain2018 done